pkg/proxy/endpointschangetracker.go

```diff
@@ -178,18 +178,6 @@ type UpdateEndpointsMapResult struct {
 	// UpdatedServices lists the names of all services with added/updated/deleted
 	// endpoints since the last Update.
 	UpdatedServices sets.Set[types.NamespacedName]
-
-	// DeletedUDPEndpoints identifies UDP endpoints that have just been deleted.
-	// Existing conntrack NAT entries pointing to these endpoints must be deleted to
-	// ensure that no further traffic for the Service gets delivered to them.
-	DeletedUDPEndpoints []ServiceEndpoint
-
-	// NewlyActiveUDPServices identifies UDP Services that have just gone from 0 to
-	// non-0 endpoints. Existing conntrack entries caching the fact that these
-	// services are black holes must be deleted to ensure that traffic can immediately
-	// begin flowing to the new endpoints.
-	NewlyActiveUDPServices []ServicePortName
```

Comment:
shouldn't this be dropping `UpdateServiceMapResult.DeletedUDPClusterIPs` too?

Reply:
Yes, will update.